components/Quiz.tsx

```diff
@@ -0,0 +1,73 @@
+"use client"
+
+import { Button } from "@shadcn/ui/button"
+import { Checkbox } from "@shadcn/ui/checkbox"
+import { Label } from "@shadcn/ui/label"
+import { RadioGroup, RadioGroupItem } from "@shadcn/ui/radio-group"
+
+import { Typography } from "./Typography"
+
+type Answer = {
+  text: string
+  isCorrect: boolean
+}
+
+interface Props {
+  question: string
+  answers: Answer[]
+}
+
+export function Quiz({ question, answers }: Props) {
+  const correctAnswers = answers.filter((answer) => answer.isCorrect).map((answer) => answer.text)
```

Comment:
This should be moved into `onSubmit` as it is only used there.

Reply:
Okay 👍🏽